docs/samples/fluid/alluxio.yaml

```diff
@@ -50,11 +49,14 @@ spec:
     levels:
       - mediumtype: SSD
         path: /mnt/ssd0/cache
-        quota: 10Gi
+        quota: 100Gi
         high: "0.95"
         low: "0.7"
   master:
     nodeSelector:
-      node.kubernetes.io/instance-type: m5.xlarge
+      node.kubernetes.io/instance-type: m5n.xlarge
+  worker:
+    nodeSelector:
+      node.kubernetes.io/instance-type: m5n.xlarge
   fuse:
     cleanPolicy: OnDemand
```

Comment:
How about  not specifying the cleanPolicy. I suggest using default policy.

Reply:
Hi @cheyang , I updated the example to use the default cleanup Policy, however for the benchmark result, I will keep it as I mainly want to compare the speed between pulling from S3 and pulling from the cache workers.